src/aura/BGE_BatchGiftEntryTab/BGE_BatchGiftEntryTab.cmp

```diff
@@ -40,7 +40,7 @@
     <aura:attribute name="modalOpen" type="Boolean" description="Prevents multiple modals from opening"/>
 
     <!-- events we handle -->
-    <aura:handler name="init" value="{!this}" action="{!c.doInit}"/>
+    <!-- <aura:handler name="init" value="{!this}" action="{!c.doInit}"/> -->
```

Comment:
Do we want to comment this markup out as opposed to deleting entirely? Not sure it's a big deal either way! Not a blocker for sure

Reply:
Yeah, I struggled with this a bit. I removed the lines that were related to the previous retirement prompt, but opted to comment out all the "functional" related lines since (as Victor rightfully reminded me) this is simply a UI work item. No functional related code should be getting removed at this point.